components/event-bus/Makefile

```diff
@@ -79,3 +79,7 @@ clean:
 	rm -rf ./cmd/event-bus-push/$(BINARY_PUSH) ./cmd/event-bus-push/docker
 	rm -rf ./cmd/event-bus-subscription-controller-knative/$(BINARY_SCK) ./cmd/event-bus-subscription-controller-knative/docker
 	rm -rf ./cmd/event-bus-sv/$(BINARY_SV) ./cmd/event-bus-sv/docker
+
+.PHONY: path-to-referenced-charts
+path-to-referenced-charts:
+	@echo "resources/event-bus"
```

Comment:
Please add a newline in the end of the file.

Reply:
:+1: